index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
```

Comment:
thanks for adding this file back in. We must have accidentally deleted it. Let's make sure we're checking our git status before we make commits.